synchronizer/l1_parallel_sync/l1_workers.go

```diff
@@ -67,7 +67,7 @@ func (w *workers) String() string {
 	return result
 }
 
-func newWorkers(ethermans []EthermanInterface, cfg workersConfig) *workers {
+func newWorkers(ethermans []L1EthermanInterface, cfg workersConfig) *workers {
```

Comment:
Why this change?? Etherman only reads info from L1

Reply:
the interface is the specific for L1Parallel, you are right that is a bad name. I'll rename to something more specific